_includes/manuals/nightly/3.3.2_software_collections.md

```diff
@@ -1,14 +0,0 @@
-The RPMs use a packaging technique called Software Collections, or SCL.  This provides Ruby and all dependencies required to run Foreman separately from the version of Ruby provided by the distribution.
```

Comment:
I can't comment on it, but that you renamed 3.3.3 to 3.3.2 must also be reflected here in https://github.com/theforeman/theforeman.org/blob/gh-pages/manuals/nightly/index.md. Right now Jekyll is unhappy.